source/adapters/native_cpu/common.cpp

```diff
@@ -0,0 +1,32 @@
+//===---------------- common.cpp - Native CPU Adapter ---------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
```

Comment:
The license headers should be updated. Can be done either as part of this PR or as a follow-up PR.

Reply:
Thought I got everything! Thanks Fabio I've updated it in this PR.